monitor.go

```diff
@@ -228,8 +266,63 @@ func getRoundRect(dc drawContext, x, y, w, h int, clr, clrPen Color) (image.Imag
 	return svg.ToImage()
 }
 
-func (p *stageMonitor) hit(hc hitContext) (hr hitResult, ok bool) {
+func (p *Monitor) hit(hc hitContext) (hr hitResult, ok bool) {
 	return
 }
 
 // -------------------------------------------------------------------------------------
+// IWidget
+func (pself *Monitor) GetName() string {
```

Comment:
Here it is `*Monitor` instead of `Monitor` who implemented interface `Widget`. Hence the user will have to do type assertion like this:

```go
scoreMonitor := getWidget("life").(*Monitor)
```

To understand the code, the user will have to know the concept "pointer", which I think is not expected. To avoid the user noticing concept "pointer", I'll suggest to rename `type Monitor struct` to something like `MonitorImpl` and make a type alias for `*MonitorImpl`:

```go
type MonitorImpl struct {}

func (pself *MonitorImpl) Visible() bool {}
func (pself *MonitorImpl) Show() {}
func (pself *MonitorImpl) Hide() {}

type Monitor = *MonitorImpl
```

Then the user can do type assertion with `Monitor` without noticing pointer:

```go
scoreMonitor := getWidget("life").(Monitor)
```

Reply:
As discussed, we will support

```go
scoreMonitor := getWidget(Monitor, "life")
```

rather than

```go
scoreMonitor := getWidget("life").(Monitor)
```